### PR TITLE
Add the Referral accepted table to ReferralDetails

### PR DESCRIPTION
**Pull request by peterhudec**
## Description of change

The _referral details page_ now shows the _Referral accepted_ section instead of the action buttons when the referral is _completed_.

## Test instructions

1. Go to the the _referral details page_ of a _completed_ referral.
2. You should not see the _Complete referral_ and _I cannot complete the referral_.
3. If you visit the same page with an _incomplete_ referral, you should still see the buttons without the _Referral accepted_ section.

## Screenshots
Screen Shot 2020-03-19 at 3 07 10 PM

### CODE REVIEW

File: src/apps/companies/apps/referrals/details/client/ReferralDetails.jsx [hunk 1]
```diff
@@ -69,26 +73,44 @@ export default connect(state2props)(
...
+                  <Button as={Link} href="/">
+                    Complete referral
+                  </Button>
+                  <SecondaryButton
+                    as={Link}
+                    href={urls.companies.referrals.help(companyId, referralId)}
+                  >
+                    I cannot complete the referral
+                  </SecondaryButton>
+                  <Link href="/">Back</Link>
```
Comment (rafenden) [inline src/apps/companies/apps/referrals/details/client/ReferralDetails.jsx]: We still should use the urls module when redirecting to the homepage.

File: src/apps/companies/apps/referrals/details/client/ReferralDetails.jsx [hunk 2]
```diff
@@ -12,6 +12,7 @@ import Task from '../../../../../../client/components/Task'
 import { state2props } from './state'
 
 import { REFERRAL_DETAILS } from '../../../../../../client/actions'
+import { interaction } from '../../../../../../../test/sandbox/routes/v3/search/interaction'
```
Comment (rafenden) [inline src/apps/companies/apps/referrals/details/client/ReferralDetails.jsx]: We shouldn't import fixtures in our production code.

File: src/apps/companies/apps/referrals/transformer.js [hunk 3]
```diff
@@ -1,3 +1,9 @@
+const convertAdviser = ({ name, contact_email, dit_team }) => ({
```
Comment (rafenden) [inline src/apps/companies/apps/referrals/transformer.js]: We usually call these transformers, i.e. `transformAdviser`

File: src/apps/companies/apps/referrals/details/client/ReferralDetails.jsx (hunk 2 above)
Comment (peterhudec) [reply src/apps/companies/apps/referrals/details/client/ReferralDetails.jsx]: Bloody autoimport 😠

**Pull request merged**